src/main/java/seedu/address/logic/parser/ItemListParser.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.parser;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.recipe.Ingredient;
+import seedu.address.model.recipe.Instruction;
+
+/**
+ * Parses and tokenises arguments to the Instruction and Ingredients command.
+ */
+public class ItemListParser {
+
+    /**
+     * Parses and returns a {@link Stream} of {@link Ingredient}s from the provided {@code itemString}.
+     * @param itemString the {@link String} to be parsed
+     * @return
```

Comment:
need to add the return value for the header comments

Reply:
Nice find @muhd97